tests/models/test_feed.py

```
@@ -0,0 +1,47 @@
+        self.assertEqual(feed.title, "Test Feed")
+        self.assertEqual(feed.url, "http://example.com")
+        self.assertEqual(feed.genre, "News")
+        self.assertEqual(feed.description, "")
+        self.assertIsNone(feed.deleted_at)
+
+        # Test instantiation with optional fields
+        feed = Feed(title="Test Feed", url="http://example.com", genre="News", description="A test feed", deleted_at=datetime(2023, 1, 1))
+        self.assertEqual(feed.description, "A test feed")
+        self.assertEqual(feed.deleted_at, datetime(2023, 1, 1))
+
+    def test_hash_property(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_hash_property`

Add return type annotation: `None`

(ANN201)